docs/migrations/v2-to-v3.md

```diff
@@ -0,0 +1,42 @@
+# Migrating from v2 to v3
+
+Oviously, start by upgrading the `cozy-konnector-libs` package.
+
+```bash
+$> yarn upgrade cozy-konnector-libs --save
+```
+
+## Deprecated attributes
+Following konnector's attributes may be removed, as they are not used anymore:
+* `color`
+
+## Removed models
+The following models has been removed and code using them has to be adapted:
+* Bill
+
+## Removed methods
+The following methods has been removed from codebase:
+* `filterExisting`
+* `linkBankOperation`
+These both ones have been directly encapsulated into `saveBills`
```

Comment:
For connectors not directly related to bills, there are saveFiles`, `filterData`, `addData` and `linkBankOperation` which are called by saveBills internally.